Library/Homebrew/cask/artifact/abstract_flight_block.rb

```diff
@@ -36,6 +36,12 @@ def summarize
         directives.keys.map(&:to_s).join(", ")
       end
 
+      def to_h
+        require "method_source"
```

Comment:
Should we add this to the gemfile if we're going to use this directly? Looking at gemfile.lock it's only added as an indirect dependency of `tapioca`.

```
tapioca -> pry -> method_source
```

Reply:
Good thinking. I just pushed what I think is the correct change. I'm not really sure how all the gem stuff works though so I might not have put it in the right spot